tests/storm/common.py

```diff
@@ -0,0 +1,113 @@
+import os
+import time
+from jinja2 import FileSystemLoader, Environment
+import locust
+from pyquery import PyQuery
+from urllib.parse import urlparse
+import gevent
+
+
+templates_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), "templates")
+template_env = Environment(loader=FileSystemLoader(templates_path), autoescape=True)
+
+
+def execute_async(client, name, identifier, inputs=None, complex_inputs=None):
```

Comment:
Add 1-line docstring just to clarify what each function does.

Reply:
done.